src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java

```diff
@@ -47,6 +47,9 @@
     // Tag thumbnail.
```

Comment:
Would be better Image thumbnail

Reply:
It is only a tag name, Is it necessary to change it?